src/all/nhentai/src/eu/kanade/tachiyomi/extension/all/nhentai/NHentai.kt

```diff
@@ -74,7 +74,7 @@ open class NHentai(
     }
 
     private val shortenTitleRegex = Regex("""(\[[^]]*]|[({][^)}]*[)}])""")
-    private val dataRegex = Regex("""JSON.parse\("([^*]*)"\)""")
+    private val dataRegex = Regex("""JSON\.parse\(\s*"([^"\\]*(?:\\.[^"\\]*)*)"\s*\)""")
```

Comment:
It's usually a bad idea to parse escape sequences with regex. Consider using something more greedy, like:
```kt
 private val dataRegex = Regex("""JSON\.parse\(\s*"(.*)"\s*\)""")
```

This completes in 26 steps, as opposed to ~2200 for some arbitrary entry.

Reply:
Yeah, that's made more sense. Thanks for the suggestion